pydantic/dataclasses.py

```diff
@@ -124,7 +125,7 @@ def dataclass(
         order: Determines if comparison magic methods should be generated, such as `__lt__`, but not `__eq__`.
         unsafe_hash: Determines if a `__hash__` method should be included in the class, as in `dataclasses.dataclass`.
         frozen: Determines if the generated class should be a 'frozen' `dataclass`, which does not allow its
-            attributes to be modified after it has been initialized.
+            attributes to be modified after it has been initialized. If not set, the value from the provided `config` argument will be used (and will default to `False` otherwise).
```

Comment:
It's the other way around - the `dataclass` argument takes priority...

Reply:
It seems the points you mentioned have already been added

Reply:
Not sure what you mean - this comment is still incorrect...

Reply:
Ah I see - this is fine.